src/RegistryCoordinator.sol

```diff
@@ -213,7 +215,7 @@ contract RegistryCoordinator is EIP712, Initializable, IRegistryCoordinator, ISo
          */
         bytes32 operatorId = blsApkRegistry.getOperatorId(msg.sender);
         if (operatorId == 0) {
-            operatorId = blsApkRegistry.registerBLSPublicKey(msg.sender, params);
+            operatorId = blsApkRegistry.registerBLSPublicKey(msg.sender, params, pubkeyRegistrationMessageHash(msg.sender));
         }
```

Comment:
Could consider tossing these lines into a helper, since we call them twice.

`bytes32 operatorId = _getOrCreateOperatorId(msg.sender, params);`

I don't feel strongly about this though.